packages/config/config/devices/0x015f/mh9-co2-wa.json

```diff
@@ -19,8 +19,45 @@
 			"label": "CO2 Notification Threshold",
 			"valueSize": 2,
 			"minValue": 1,
-			"maxValue": 2000,
+			"maxValue": 5000,
 			"defaultValue": 1000
+		},
+		"2": {
+			"label": "CO2 reporting threshold",
+			"description": "0 report disabled, base on 5ppm unit, 10 by default, 10*5ppm=50ppm",
```

Comment:
Please don't mention units or default values in the description. We have the `"unit"` and `"defaultValue"` properties for that. Same goes for the other parameters.

Reply:
The special meaning of `0 = report disabled` should be added as a single entry under `"options"` instead. Those are used as suggestions in UIs.